docs/content/en/docs/faqs/_index.md

```diff
@@ -87,7 +87,25 @@ helm install demo k8ssandra/k8ssandra -n k8ssandra --create-namespace
 
 ### Can I install multiple releases of K8ssandra?
 
-Some of the objects installed by the K8ssandra chart are currently configured to be cluster-scoped; consequently, you should only install those components once. This should be fixed before version 1.0 to allow multiple installations. Other parts can be installed multiple times to allow creating multiple Cassandra clusters in a single k8s cluster.
+You can install multiple releases of K8ssandra in a Kubernetes environment, provided:
+
+* You wait for all the pods to reach a Ready or Completed state for a given K8ssandra install, before attempting the next K8ssandra install
```

Comment:
I didn't know this. What happens if you attempt to install another before then?

Reply:
I thought K8ssandra is like a super operator over cass-operator. I didn't know that for installing another Cassandra cluster, I'll have to install the whole stack again.

Reply:
@slayerjain k8ssandra is, for now at least, not an operator, but a set of Helm charts. It greatly simplifies the process of configuring cass-operator by establishing sane defaults. It also helps you configure the other components you need in order to have a production-ready Cassandra database: backup/restore, repair, coordinator-only nodes for offloading compute work from storage nodes (aka Stargate), ingress, monitoring, and so on.

If you want multiple Cassandra clusters in your Kubernetes cluster, then yes, you need to manage a values.yaml (or, if you're a masochist, a set of `--set` arguments) and invoke `helm install` once for each.  

I'm not sure of the current status of this effort (post-1.0 planning is happening today), but the plan is that some components (e.g. monitoring) can be shared across k8ssandra clusters.

Reply:
@jakerobb thanks for the detailed explanation.